src/main/js/portal/main/models/Preview.ts

```diff
@@ -60,17 +58,29 @@ export default class Preview {
 		this.pids = this.items.map(item => getLastSegmentInUrl(item.dobj));
 		this.options = options ?? [];
 		this.type = type;
-		this.previewSettings = this.item?.urlParams ? Preview.allowlistPreviewSettings(this.item.urlParams) : {};
+		this.previewSettings = Preview.parsePreviewSettings(this.item?.urlParams || {});
```

Comment:
```suggestion
		this.previewSettings = Preview.parsePreviewSettings(this.item?.urlParams ?? {});
```

In general I think the nullish coalescing operator is better for this kind of expression.